src/Engine/EmitMSIL/CodeGenIL.cs

```diff
@@ -1172,11 +1172,11 @@ private Type EmitFunctionCallNode(AssociativeNode node)
                         return indexResult.type;
                     }
                     //if we fail to emit indexing at compile time, 
-                    //emit a function call to ValueAtIndexDynamic to avoid overload issues.
+                    //emit a function call to ValueAtIndex.
                     else
                     {
-                        methodName = nameof(DesignScript.Builtin.Get.ValueAtIndexDynamic);
-                        EmitILComment("NOT ENOUGH TYPE INFO TO EMIT INDEXING, EMIT VALUEATINDEXDYNAMIC FUNCTION CALL");
+                        methodName = nameof(DesignScript.Builtin.Get.ValueAtIndex);
+                        EmitILComment("NOT ENOUGH TYPE INFO TO EMIT INDEXING, EMIT ValueAtIndex FUNCTION CALL");
```

Comment:
hmm, thats okay with me, I hope we can implement compile time function calls without replication soon though.

Reply:
In that case, can we remove the implementation for `ValueAtIndexDynamic` form `BuiltIn.cs`?

Reply:
@pinzart90, won't we have the same issue with the builtin operators that @sm6srw added in this case, where we are replicating over those operators?

Reply:
yes we should remove `ValueAtIndexDynamic`.
If @sm6srw 's builtin operators are only supposed to handle rank 0 arguments...then all should be fine. If you pass it a list of args, then it will (and should) replicate.

I did not see any arrays or lists to be handled in those builtin operators

Reply:
Ok, so as long as the parameter is supposed to be a single value and is of type object, it's fine, got it.